lib/data/sql/V3__create_types.sql

```diff
@@ -834,6 +834,45 @@ FROM nat_boden
 LEFT JOIN nat_naistyp ON nat_boden.naistyp_sort = nat_naistyp.naistyp_sort
 WHERE trim(nat_naistyp.naistyp_c) = foresttype_meta.code;
 
+ALTER TABLE lu_standorttypen
+ADD expoAndAspect INTEGER[];
+
+UPDATE lu_standorttypen
```

Comment:
I would try to avoid messing around with imported data like this, because it will be difficult to trace down where some data is coming from.

Better join this data to the export query in V5.

Reply:
I moved all of this into export_projections